db/migrate/20170524195528_create_problems.rb

```diff
@@ -3,7 +3,6 @@ def change
     create_table :problems do |t|
       t.string :name
       t.text :description
-      t.string :cmd
```

Comment:
Going forward, we only need a single templated command to run submissions per programming language, parameterized by path of the test input file and possibly a canon solution.